rest-fights/src/test/java/io/quarkus/sample/superheroes/fight/DisableIfContinuousTesting.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkus.sample.superheroes.fight;
+
+import io.quarkus.sample.superheroes.fight.DisableIfContinuousTesting.ContinuousTestingCondition;
+
+import org.junit.jupiter.api.extension.ConditionEvaluationResult;
+import org.junit.jupiter.api.extension.ExecutionCondition;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.extension.ExtensionConfigurationException;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+@ExtendWith(ContinuousTestingCondition.class)
+public @interface DisableIfContinuousTesting {
+
+    class ContinuousTestingCondition implements ExecutionCondition {
+      @Override
+      public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext context) {
+        final Class clazz = context.getRequiredTestClass();
```

Comment:
Why declare the type `Class` on this line but use `var` on the next? Any particular reason?

Reply:
Copy and pasting :)